### PR TITLE
Update and improvements.

### PR DESCRIPTION
**Pull request by amerkoleci**
- Update Vortice.Windows to latest 2.1.0 stable release.
- Install .NET SDK 6.0 in CI build for future .NET 6.0 update.
- Cleanup dependencies.

### CODE REVIEW

**Comment by amerkoleci** [comment]
Something was broken in Compute shader logic in Vortice.Direct3D11 thats causes tests to fail (like BasicCompute), seams something related to Map or CS binding logic (zzz)

File: src/NeoDemo/NeoDemo.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```
Comment (TechnologicalPizza) [inline src/NeoDemo/NeoDemo.csproj:5]: I don't see why we would jump to net5 when net6 is the latest LTS.

Comment (amerkoleci) [reply src/NeoDemo/NeoDemo.csproj:5]: It was not part of this PR, but I can submit .NET 6.0 migration PR as well :D

**Comment by TechnologicalPizza** [comment]
These failures may be caused by `BufferDescription.RawBuffer` being false by default.

**Pull request merged**